addon/macro.js

```diff
@@ -3,29 +3,72 @@
  * https://github.com/jamesarosen/ember-i18n/blob/master/addon/utils/macro.js
  */
 import { assert } from '@ember/debug';
-import { computed, get } from '@ember/object';
+import { computed, get, getProperties } from '@ember/object';
 import { getOwner } from '@ember/application';
 import EmptyObject from 'ember-intl/-private/empty-object';
 
-const { keys } = Object;
+function partitionDynamicValuesAndStaticValues(options) {
+  const dynamicValues = new EmptyObject();
+  const staticValues = new EmptyObject();
 
-function values(object) {
-  return keys(object).map(key => object[key]);
+  Object.keys(options).forEach(key => {
```

Comment:
Intentionally using this old clunky syntax instead of `for (const [k, v] of Object.entries(...))` here to avoid transpilation bloat for ancient browsers.